requirements.txt

```diff
@@ -0,0 +1,4 @@
+black
+bandit
+flake8
+pre-commit
```

Comment:
Shouldn't we specify the same versions that are given in the file linter-requirements-txt?

Reply:
I prefer not. Actually we should be using the latest ones. I think this is something that should be improved in `linter-requirements.txt` too